CODE_OF_CONDUCT.md

```diff
@@ -1,7 +1,80 @@
- 3. **We do not allow** inappropriate or otherwise unacceptable behavior, such as sexualized language, trolling and insulting comments, or otherwise promoting intolerance or harassment. Publishing the personal information of others without their consent is prohibited.
- 4. **We are responsible** for clarifying the standards of our community, and have the right to remove or alter the comments of those participating within our community, as necessary.
- 5. Our Code of Conduct applies within all project spaces, as well as when an individual is representing the Privacy Guides project in other communities.
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+-   Using welcoming and inclusive language
+-   Being respectful of differing viewpoints and experiences
+-   Gracefully accepting constructive criticism
+-   Focusing on what is best for the community
+-   Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+-   The use of sexualized language or imagery and unwelcome sexual
+    attention or advances
+-   Trolling, insulting/derogatory, antagonistic comments and personal
+    or political attacks
+-   Promoting intolerance
+-   Public or private harassment
+-   Publishing others\' private information, such as a physical or
+    electronic address, without explicit permission
+-   Other conduct which could reasonably be considered inappropriate in
+    a professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of
+acceptable behavior and are expected to take appropriate and fair
+corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit,
+or reject comments, commits, code, wiki edits, issues, and other
+contributions that are not aligned to this Code of Conduct, or to ban
+temporarily or permanently any contributor for other behaviors that they
+deem inappropriate, threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies within all project spaces, and it also
+applies when an individual is representing as a part of the project or
+its community in public spaces.
+
+Examples of representing a project or community include using an
+official project e-mail address, posting via an official social media
+account, or acting as an appointed representative at an online or
+offline event. Representation of a project may be further defined and
+clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may
+be reported by contacting the *Internal Compliance Officer*. The report should
+include information on whether the details can be shared with other
+*Compliance Team* and *Core Team* Members and how much may be
```

Comment:
These *emphatised* parts used to be links, but I have no idea where they were going to, so I just turned them like this as a note to fix them or discuss who these people are.